doc/running.rst

```diff
@@ -0,0 +1,25 @@
+Running MirgeCOM
+================
+
+Running with large numbers of ranks and nodes
+---------------------------------------------
+
+The PyOpenCL package used in MirgeCOM caches results of compilation on the hard
+disk in order to speed up multiple executions of the same kernel. This can lead
+to slowdowns on startup when executing on many ranks and/or nodes due to concurrent
+hard disk accesses. We have seen these slowdowns starting with 256 ranks/nodes.
+An indicator of concurrency issues are warnings like these:
+
+.. code::
+
+  .conda/envs/dgfem/lib/python3.8/site-packages/pyopencl/cache.py:101: UserWarning:
+  could not obtain cache lock--delete '.cache/pyopencl/pyopencl-compiler-cache-v2-py3.8.3.final.0/lock' if necessary
```

Comment:
4-wide indent (below too)

Reply:
Isn't the offical indentation for rst 3 spaces? Changed to 3.